org.guerinoni.qTsConverter.yml

```diff
@@ -0,0 +1,20 @@
+app-id: org.guerinoni.qTsConverter
+runtime: org.kde.Platform
+runtime-version: '5.14'
+sdk: org.kde.Sdk
+command: qTsConverter
+finish-args:
+  - --device=dri
+  - --filesystem=host
```

Comment:
What does it need host permissions for?

Reply:
At first glance, it's something that should work fine-ish with portals, as you don't access full directories, just specific files.